### PR TITLE
Landing page updates 1 14

### PR DESCRIPTION
**Pull request by anthonyemberley**
no rush but would love a review + pull here

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
   class="style-black-1 custom-colors-enabled custom-primary-button-color-bright custom-secondary-button-color-bright custom-accent-color-bright custom_fonts comps live_website"
 >
   <head>
+    <!-- Google Optimize -->
+    <script src="https://www.googleoptimize.com/optimize.js?id=OPT-TJC99HZ"></script>
```
Comment (jaredhanson11) [inline index.html:7]: what is this, are there any risks involved here?

File: index.html [hunk 2]
```diff
@@ -471,6 +275,21 @@ <h2 class="steps-01__heading">Talk with our experts</h2>
...
+                <p>
+                  Text us at
+                  <span
+                    style="
+                      color: rgb(29, 28, 29);
+                      background-color: rgb(248, 248, 248);
+                      font-size: 15px;
+                      font-family: Slack-Lato, appleLogo, sans-serif;
+                    "
+                    ><a href="tel:415-612-6569">(415) 612-6569</a></span
```
Comment (jaredhanson11) [inline index.html:289]: nice (what number is this btw)

**Review by jaredhanson11 — APPROVED** [review]
LGTM, I pretty much just trust that you're testing all of this stuff thought so please make sure you've tested everything locally before I merge and deploy. Once I get the confirmation there I will deploy this

File: index.html (hunk 1 above)
Comment (anthonyemberley) [reply index.html:7]: this allows us to do A/B tests on the landing page with no additional changes on our end -- built by Google and shouldn't have any risks until we actually set up a test (similar to google tag manager or google analytics and all of them work together)

File: index.html (hunk 2 above)
Comment (anthonyemberley) [reply index.html:289]: just a number that will text our slack if anyone wants to reach us directly that way. Figured we would give it a try

**Comment by anthonyemberley** [comment]
Triple checked everything just now and we should be good to go!

**Pull request merged**